fuzzers/005-tilegrid/Makefile

```diff
@@ -49,8 +49,11 @@ endif
 
 # Kintex7 only fuzzers
 ifeq (${XRAY_DATABASE}, kintex7)
+# These kintex parts give an empty design
+ifneq (${XRAY_PART}, $(filter ${XRAY_PART}, xc7k160tffg676-2 xc7k325tffg900-2 xc7k480tffg1156-2))
```

Comment:
given that there are potentially many parts corresponding to the same device (e.g. xc7k160t can have a lot of parts with different pin mappings), it might be worth using the `${XRAY_FABRIC}` variable and filter on the fabric rather than the parts.

Reply:
OK done